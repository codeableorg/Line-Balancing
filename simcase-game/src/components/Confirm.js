/** @jsx jsx */
import React from "react";
import { jsx } from "@emotion/core";
import { Dialog } from "@reach/dialog";

import NextScenario from "./NextScenario";
import LastScenario from "./LastScenario";

const dialog = {
  margin: "auto",
  marginTop: "20px",
  width: "90vw",
  height: "95vh",
  padding: "0"
};

function Confirm(props) {
  const { openSubmit, closeSubmit, confirm, scenario, preFeedback } = props;

  return (
    <>
      <Dialog isOpen={confirm} onDismiss={openSubmit} css={dialog}>
        {scenario === 4 ? (
          <LastScenario
            closeSubmit={closeSubmit}
            scenario={scenario}
            preFeedback={preFeedback}
          />
        ) : (
          <NextScenario
            closeSubmit={closeSubmit}
            scenario={scenario}
            preFeedback={preFeedback}
          />
        )}
      </Dialog>
    </>
  );
}

export default Confirm;
