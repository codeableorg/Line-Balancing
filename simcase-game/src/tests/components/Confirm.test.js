import React from "react";
import { render } from "@testing-library/react";

import Confirm from "../../components/Confirm";

test("render Confirm", () => {
  const { asFragment } = render(<Confirm />);
  expect(asFragment()).toMatchSnapshot();
});
