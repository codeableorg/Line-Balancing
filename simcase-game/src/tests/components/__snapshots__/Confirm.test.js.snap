// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render Confirm 1`] = `<DocumentFragment />`;
